src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.epics.js

```diff
@@ -12,75 +12,116 @@ import { actionCreator } from '../../../../actions/actions.utils';
 
 const trackerCaptureAppUrl = () => (process.env.REACT_APP_TRACKER_CAPTURE_APP_PATH || '..').replace(/\/$/, '');
 
-export const onScopeProgramFindUsingUniqueIdentifierEpic = (action$: InputObservable, store: ReduxStore) =>
+const filtersForUniqueIdSearchQuery = (searchTerm) => {
```

Comment:
NIT: maybe `getFiltersForUniqueIdSearchQuery` for the function name? And replace `searchTerm` with something else because it's the formValues/formData object?

Reply:
doing it